app/store/networkDrawer.ts

```diff
@@ -0,0 +1,51 @@
+import { CTransactionSegWit } from '@defichain/jellyfish-transaction'
+import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+export interface Transaction {
+  broadcasted: boolean
+  signed: CTransactionSegWit
+  title?: string
```

Comment:
```suggestion
  title?: string
  err?: Error
```

I think will be better to keep for each job, possible to retry too.

Reply:
I'll keep this for now. I don't see the reason of storing the error per transaction since we only display one at a time. Currently, there's one error on OceanInterface.